src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfig.java

```diff
@@ -23,25 +23,26 @@
 @ComponentScan("uk.gov.companieshouse.api.interceptor")
 public class InterceptorConfig implements WebMvcConfigurer {
 
-    static final String COMPANY_NUMBER = "/confirmation-statement/company/**";
-    static final String TRANSACTIONS = "/transactions/**";
+    static final String COMPANY_NUMBER = "/**/company/**";
+    static final String TRANSACTIONS = "/**/transactions/**";
+    static final String PRIVATE = "/private/**";
     static final String FILINGS = "/private/**/filings";
     static final String SUBMISSIONS = TRANSACTIONS + "/confirmation-statement/{confirmation_statement_id}/**";
     private static final String NEXT_MADE_UP_TO_DATE = "/confirmation-statement/**/next-made-up-to-date";
     private static final String ELIGIBILITY = "/confirmation-statement/**/eligibility";
     private static final String COSTS = TRANSACTIONS + "/costs";
 
     static final String[] USER_AUTH_ENDPOINTS = {
-            NEXT_MADE_UP_TO_DATE,
-            ELIGIBILITY
+        NEXT_MADE_UP_TO_DATE,
+        ELIGIBILITY
     };
     static final String[] INTERNAL_AUTH_ENDPOINTS = {
-            FILINGS,
-            COSTS
+        PRIVATE,
+        COSTS
```

Comment:
Does it make sense that FILINGS and COST are "internal auth" but only FILINGS has /private as base URL?

Reply:
I have already raised this (specifically the costs endpoint) as a follow-up question, but changing the existing API is not in scope for this piece of work.

Reply:
OK, may be add a comment to the code to help readers in future